source/linux/Foundational_Components/U-Boot/UG-QSPI.rst

```diff
@@ -1,32 +1,232 @@
+   +=============+============+====================================+
+   | AM62x SK    | OSPI NOR   | :file:`drivers/spi/cadence_qspi.c` |
+   +-------------+------------+------------------------------------+
+   | AM62x LP SK | OSPI NAND  | :file:`drivers/spi/cadence_qspi.c` |
+   +-------------+------------+------------------------------------+
+
+.. ifconfig:: CONFIG_part_variant in ('AM62AX')
+
+   +------------+------------+------------------------------------+
+   | SoC Family | Capability | Driver                             |
+   +============+============+====================================+
+   | AM62Ax     | OSPI NAND  | :file:`drivers/spi/cadence_qspi.c` |
+   +------------+------------+------------------------------------+
+
+.. ifconfig:: CONFIG_part_variant in ('AM62PX', 'J7200')
+
+   +--------------+------------+------------------------------------+
+   | SoC Family   | Capability | Driver                             |
+   +==============+============+====================================+
+   | AM62Px/J7200 | OSPI NOR   | :file:`drivers/spi/cadence_qspi.c` |
+   +--------------+------------+------------------------------------+
+
+.. ifconfig:: CONFIG_part_variant in ('J721E')
+
+   +-------------+--------------+------------------------------------+
+   | SoC Family  | Capability   | Driver                             |
+   +=============+==============+====================================+
+   | AM654/J721e | 1x QSPI NOR, | :file:`drivers/spi/cadence_qspi.c` |
+   |             | 1x OSPI NOR  |                                    |
+   +-------------+--------------+------------------------------------+
+
+.. ifconfig:: CONFIG_part_variant in ('J722S')
+
+   +------------+--------------+------------------------------------+
+   | SoC Family | Capability   | Driver                             |
+   +============+==============+====================================+
+   |            | 1x OSPI NOR  | :file:`drivers/spi/cadence_qspi.c` |
+   + J722S      +--------------+------------------------------------+
+   |            | 1x OSPI NAND | :file:`drivers/spi/cadence_qspi.c` |
+   +------------+--------------+------------------------------------+
+
+.. ifconfig:: CONFIG_part_variant in ('J721S2', 'J784S4','J742S2')
+
+   +---------------+--------------+------------------------------------+
+   | SoC Family    | Capability   | Driver                             |
+   +===============+==============+====================================+
+   |               | 1x QSPI NOR, | :file:`drivers/spi/cadence_qspi.c` |
+   | J721S2/J784S4 | 1x OSPI NOR  |                                    |
+   +               +--------------+------------------------------------+
+   |               | 1x OSPI NAND | :file:`drivers/spi/cadence_qspi.c` |
+   +---------------+--------------+------------------------------------+
 
 .. note::
 
-    The sf command is used to access SPI NOR flash, supporting read/write/erase and
```

Comment:
Should we continue to put this note here ?

Reply:
The purpose of 'sf' or 'mtd' command is well understandable from the examples provided. Explaining them looks redundant.